### PR TITLE
MBS-1057: remove testType

### PR DESCRIPTION
**Pull request by dsvoronin**
deprecate separate types(used for filtering only now)
introduce `@Kind` to replace `@Type` and separate types

ignore `@Features`

Kind -> TestKind to avoid name clash, because both `@Kind` annotation and Kind enum will be used in test class

### CODE REVIEW

File: subprojects/android-test/test-annotations/src/main/kotlin/com/avito/android/test/annotations/TestType.kt [hunk 1]
```diff
@@ -1,40 +1,46 @@
 package com.avito.android.test.annotations
 
+@Deprecated("2020.3.2; use @Kind")
```
Comment (sergioboi) [inline subprojects/android-test/test-annotations/src/main/kotlin/com/avito/android/test/annotations/TestType.kt]: Maybe we need to use `replaceWith` argument?

File: subprojects/android-test/test-annotations/build.gradle.kts [hunk 2]
```diff
@@ -3,3 +3,7 @@ plugins {
     `maven-publish`
     id("com.jfrog.bintray")
 }
+
+dependencies {
+    implementation(project(":subprojects:common:report-viewer"))
```
Comment (sergioboi) [inline subprojects/android-test/test-annotations/build.gradle.kts:8]: It's not clear why do we need report-viewer in test-annotations module.
Maybe we have problems with module naming?

**Review by sergioboi — APPROVED** [review]

**Review by int02h — APPROVED** [review]

File: subprojects/android-test/test-annotations/build.gradle.kts (hunk 2 above)
Comment (dsvoronin) [reply subprojects/android-test/test-annotations/build.gradle.kts:8]: yep, `report-viewer` -> `report` and annotations should be there

File: subprojects/android-test/test-annotations/src/main/kotlin/com/avito/android/test/annotations/TestType.kt (hunk 1 above)
Comment (dsvoronin) [reply subprojects/android-test/test-annotations/src/main/kotlin/com/avito/android/test/annotations/TestType.kt]: i decided to revert `@Kind` annotation for now, and use old ones, so can't determine replaceWith value

**Pull request merged**